src/main/java/bhs/devilbotz/commands/DriveSetDistancePID.java

```diff
@@ -28,7 +28,7 @@ public DriveSetDistancePID(DriveTrain drivetrain, double distance) {
     this.drivetrain = drivetrain;
     this.distance = distance;
     distancePid =
-        new PIDController(Constants.DISTANCE_P, Constants.DISTANCE_I, Constants.DISTANCE_K);
+        new PIDController(Constants.DISTANCE_P, Constants.DISTANCE_I, Constants.DISTANCE_D);
```

Comment:
This should use `Robot.getDriveTrainConstant("DISTANCE_X")`, or we can delete this file altogether since `DriveStraightPID` can replace the functionality of `DriveSetDistance`.